XmlFormatter/src/Settings/Adapter/PropertyAdapter.cs

```diff
@@ -0,0 +1,130 @@
+        /// </summary>
+        public string Name => name;
+
+        /// <summary>
+        /// Create a new instance of this adapter which will always use "Default" as scope name
+        /// </summary>
+        public PropertyAdapter()
+        {
+            SetName("Default");
+        }
+
+        /// <summary>
+        /// Set the name of this adapter
+        /// </summary>
+        /// <param name="name">The new name to use</param>
+        public void SetName(string name)
+        {
+            this.name = name;
+        }
+
+        /// <summary>
+        /// Add a new setting to the resource file.
+        /// This will only work if the setting is already defined
+        /// </summary>
+        /// <param name="setting">The setting to add</param>
+        public void AddSetting(ISettingPair setting)
+        {
+            try
+            {
+                Type type = Properties.Settings.Default[setting.Name].GetType();
+                if (type == setting.Type)
+                {
+                    Properties.Settings.Default[setting.Name] = setting.Value;
+                }
+            }
+            catch (Exception)
+            {
+
+            }
+        }
+
+        /// <summary>
+        /// Get a setting from the resource file
+        /// </summary>
+        /// <param name="name">The name of the setting to get</param>
+        /// <returns>A instance of the setting or null if nothing was present</returns>
+        public ISettingPair GetSetting(string name)
+        {
+            ISettingPair returnValue = null;
+            try
+            {
+                var propertyValue = Properties.Settings.Default[name];
+                returnValue = new SettingPair(name);
+                returnValue.SetValue(propertyValue);
+            }
+            catch (Exception)
+            {
+            }
+            
+            return returnValue;
+        }
+
+        /// <summary>
+        /// Get all the settings stored in the resource file
+        /// </summary>
+        /// <returns>All the settings present in the resource file</returns>
+        public List<ISettingPair> GetSettings()
+        {
+            List<ISettingPair> settingPairs = new List<ISettingPair>();
+            foreach (SettingsProperty property in Properties.Settings.Default.Properties)
+            {
+                if (property.Attributes.Contains(typeof(ApplicationScopedSettingAttribute)))
+                {
+                    continue;
+                }
+                SettingPair settingPair = new SettingPair(property.Name);
+                settingPair.SetValue(Properties.Settings.Default[property.Name]);
+                settingPairs.Add(settingPair);
+            }
+            return settingPairs;
+        }
+
+        /// <summary>
+        /// This method is not working for this adapter
+        /// </summary>
+        /// <param name="scope">The scope to add</param>
+        public void AddSubScope(ISettingScope scope)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5052706)